_overviews/index.md

```diff
@@ -6,6 +6,7 @@ languages: [ja, zh-cn, ru, uk]
 permalink: /overviews/:title.html
 redirect_from:
   - /scala3/guides.html
+  - /guides.html
```

Comment:
Why do we add this redirection?

Reply:
because there existed `guides.md` that previously redirected, but I deleted it